### PR TITLE
Feat/encode eth signature

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Compatible with https://github.com/cennznet/cennznet/pull/639

### CODE REVIEW

**Comment by KarishmaBothara** [comment]
the test will pass, only when we update with new docker image from node with the changes

File: packages/types/src/interfaces/extrinsic/v1/Extrinsic.ts [hunk 1]
```diff
@@ -95,8 +96,9 @@ export default class CENNZnetExtrinsic extends GenericExtrinsic{
     const cennznetAddress = cvmToAddress(ethAddress);
     const nonce = await api.rpc.system.accountNextIndex(cennznetAddress);
     const payload = this.registry.createType('EthWalletCall', { call: this, nonce }).toHex();
+    const encodedPayload = `data:application-octet;base64,${base64Encode(payload)}`;
```
Comment (ken-futureverse) [inline packages/types/src/interfaces/extrinsic/v1/Extrinsic.ts]: should be `data:application/octet-stream`

**Review by ken-futureverse — CHANGES_REQUESTED** [review]

File: packages/types/src/interfaces/extrinsic/v1/Extrinsic.ts (hunk 1 above)
Comment (KarishmaBothara) [reply packages/types/src/interfaces/extrinsic/v1/Extrinsic.ts]: ya was waiting for image from node changes.. updated prefix and signature with change in prefix

File: packages/api/src/util/helper.ts [hunk 2]
```diff
@@ -48,15 +50,25 @@ export async function awaitDepositClaim(
...
+            } else if (
+              (section === 'system' && method === 'ExtrinsicFailed') ||
+              section === 'erc20Peg' ||
+              method === 'DelayedErc20DepositFailed'
+            ) {
               reject('Claim already notarized');
             }
           }
         }
+        console.log('event claim id:', eventClaimId);
```
Comment (ken-futureverse) [inline packages/api/src/util/helper.ts]: remove the `console.log`?

**Review by ken-futureverse — APPROVED** [review]

**Pull request merged**